appstore/developer_portal_api.py

```diff
@@ -249,7 +245,11 @@ def update_app_fields(app_id):
         # Check title length
         if "title" in req and len(req["title"]) > 45:
             return jsonify(error="Title must be less than 45 characters", e="invalid.field.title"), 400
-            
+
+        # Create a timeline token when enabling the timeline
+        if "timeline_enabled" in req:
+            req["timeline_token"] = secrets.token_urlsafe(32) if req["timeline_enabled"] else None
```

Comment:
I *think* this will create a new timeline_token even for apps that already have one, right?

Reply:
Hm, yes, that's a fair point